src/hooks/dialogmote/useInitialValuesReferat.ts

```diff
@@ -19,6 +20,8 @@ export const useInitialValuesReferat = (
     if (!referat) {
       return {
         naermesteLeder: currentNarmesteLederNavn,
+        behandlerDeltatt: behandler?.deltatt ?? true,
+        behandlerMottarReferat: behandler?.mottarReferat ?? true,
```

Comment:
Er default-verdi true er en midlertidig greie inntil det er kommer mer kode senere?

Reply:
Nei, det er nok egentlig ikke meningen. Har for så vidt ingen betydning hva disse settes til når behandler ikke er med, men enig at det er litt misvisende. Kan fikse!